doc/reference/reference_lua/connpool.rst

```diff
@@ -0,0 +1,188 @@
+..  _connpool_module:
+
+Module experimental.connpool
+============================
+
+**Since:** :doc:`3.1.0 </release/3.1.0>`
+
+..  important::
+
+    ``experimental.connpool`` is an experimental module and is subject to changes.
+
+An ``experimental.connpool`` module provides a set of features for connecting to remote cluster instances and for executing remote procedure calls on an instance that satisfies the specified criteria.
+
+..  NOTE::
+
+    Note that the execution time for ``experimental.connpool`` functions depends on the number of instances and the time required to connect to each instance.
+
+
+
+..  _connpool_module_load:
+
+Loading a module
+----------------
+
+To load the ``experimental.connpool`` module, use the ``require()`` directive:
+
+..  code-block:: lua
+
+    local connpool = require('experimental.connpool')
+
+
+..  _connpool_module_api_reference:
+
+API Reference
+-------------
+
+..  container:: table
+
+    ..  rst-class:: left-align-column-1
+    ..  rst-class:: left-align-column-2
+
+    ..  list-table::
+        :widths: 35 65
+
+        *   -   **Functions**
+            -
+
+        *   -   :ref:`connpool.call() <connpool_module_call>`
+            -   Execute the specified function on a remote instance
+
+        *   -   :ref:`connpool.connect() <connpool_module_connect>`
+            -   Create a connection to the specified instance
+
+        *   -   :ref:`connpool.filter() <connpool_module_filter>`
+            -   Get names of instances that match the specified conditions
+
+
+
+
+
+.. module:: connpool
+
+..  _connpool_module_api_reference_functions:
+
+Functions
+~~~~~~~~~
+
+..  _connpool_module_call:
+
+..  function:: call(func_name, args, opts)
+
+    Execute the specified function on a remote instance.
+
+    ..  NOTE::
+
+        You need to grant the ``execute`` :ref:`permission <configuration_credentials_managing_users_roles_granting_privileges>` for the specified function to a user used for replication to execute this function on a remote instance.
```

Comment:
```suggestion
        You need to grant the ``execute`` :ref:`permission <configuration_credentials_managing_users_roles_granting_privileges>` for the specified function to users with the ``replication`` :ref:`role <box_space-user>` to execute this function on a remote instance.
```

Reply:
Technically speaking, a _user used for replication_ and a _user with the replication role_ might be different entities. In particular, _user used for replication_ maintains inter-cluster communications while a _user with the replication role_ can be created to allow some third-party system get data from Tarantool cluster (for example, CDC).